Orange/classification/naive_bayes.py

```diff
@@ -48,10 +48,10 @@ def __init__(self, log_cont_prob, class_prob, domain):
         self.class_prob = class_prob
 
     def predict_storage(self, data):
+        if isinstance(data, Instance):
+            data = Table(np.atleast_2d(data.x))
```

Comment:
Thanks. I somehow thought I've done this correctly. (But I have no good explanation for `assert_not_called` in tests. :))